pkg/didcomm/protocol/introduce/metadata.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package introduce
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/service"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/didexchange"
+)
+
+const (
+	metaContextID    = "context_id"
+	metaSkipProposal = "skip_proposal"
+	metaInvitation   = "invitation"
+	metaRecipients   = "recipients"
+)
+
+// Opt describes option signature for the Continue function
+type Opt func(m map[string]interface{})
+
+// WithInvitation is used when introducee wants to provide invitation.
+// NOTE: Introducee can provide invitation only after receiving ProposalMsgType
+// USAGE: event.Continue(WithInvitation(inv))
```

Comment:
How would it be called from an async client like REST or WASM that cannot call event.Continue?

Reply:
ie, there needs to be a method on Introduce that can continue the state machine based on a serialized identifier (presumably the thread ID). we cannot rely on the event object instance as the continuation might be called on the REST/WASM api, after restart or even on a different cluster node.

Reply:
Done! I have exposed the `Continue` function for that.